packages/dita-example-sx-modules-xsd-reference-mod/src/api/removePropertiesColumnCustomMutation.js

```diff
@@ -26,11 +27,7 @@ export default function removePropertiesColumn(stepData, blueprint) {
 
 	// Query the relevant node names in all rows of the same properties table
 	var columnNodes = evaluateXPathToNodes(
-		'parent::*/parent::properties/*/*[self::' +
-			contextNodeName +
-			' or self::' +
-			otherNodeName +
-			']',
+		xq`parent::*/parent::properties/*/*[self::${contextNodeName} or self::${otherNodeName}]`,
```

Comment:
This will need to be interpolated differently. I'd suggest this:

```
parent::*/parent::properties/*/*[name() = ${contextNodeName} or name() = ${otherNodeName}]
```